### ISSUE TITLE
CLI entry point regression

### ISSUE THREAD

**Issue by dokempf** [eriwen/lcov-to-cobertura-xml#40]
Between `1.6` and `2.0` there has been a regression w.r.t. the command line entry point. Doing `python -m lcov_cobertura` would work in 1.6, but produce the following error in 2.0:

```
../bin/python: No module named lcov_cobertura.__main__; 'lcov_cobertura' is a package and cannot be directly executed
```

Most probably an accidental side effect of the `setup.py` -> `setup.cfg` transition.

**Comment by sarnold** [comment]
Yes, you cannot import a package name, and in this case you get *exactly the same behavior* if you try importing the module name:

```
python -m lcov_cobertura.lcov_cobertura

    Converts LCOV coverage data to Cobertura-compatible XML for reporting.

    Usage:
        lcov_cobertura.py lcov-file.dat
        lcov_cobertura.py lcov-file.dat -b src/dir -e test.lib -o path/out.xml

    By default, XML output will be written to ./coverage.xml
```

**Comment by eriwen** [comment]
Fixed by a07765117c23073e36cfd2e88e3793c6ae57045d and released in version 2.0.2 https://pypi.org/project/lcov-cobertura/2.0.2/
